operator/operator.go

```diff
@@ -172,9 +172,16 @@ func (operator *Operator) Serve(port uint16) error {
 
 	operator.rpcOperator = NewRPCOperator(operator.enclave)
 
+	ctxReq, cancel := eth.ContextNodeReq()
+	defer cancel()
```

Comment:
You could move this into the new `NetworkID` method and remove the context from its parameters.